src/Textarea/Textarea.stories.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {text, boolean} from '@storybook/addon-knobs';
+
+import {Container} from './Container';
+import {Textarea} from './Textarea';
+
+storiesOf('Textarea', module)
+  .addWithJSX('default', () => (
+    <Textarea
+      id="textarea_id"
+      label={text('label', 'Textarea label')}
+      value={text('value', '')}
+      resize="both"
+      onChange={() => {}}
+      disabled={boolean('disabled', false)}
+      placeholder="Here is some placeholder text."
+      className={text('Class name', '')}
```

Comment:
this **props** is required by `styled-components` only
I don't think we would have any use of explicitly setting it in the storybook